template_rpm/distribution.sh

```diff
@@ -37,7 +37,12 @@ if [ -z "${DIST_VER}" ]; then
     error "Please provide DIST_VER in environment."
 fi
 
-DNF_OPTS=(-y --releasever "${DIST_VER}")
+DNF_OPTS=(-y
+          "--releasever=${DIST_VER}"
+          --setopt=deltarpm=False
+          --setopt=zchunk=False
+          --setopt=gpgcheck=1
+          --setopt=localpkg_gpgcheck=1)
```

Comment:
This reliably breaks building any non-minimal template, because it prevents installing package that ships rpmfusion key. Don't do that.